pkgs/applications/misc/inherd-quake/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, stdenv
+, CoreServices
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "quake";
```

Comment:
maybe this name should also be changed?

Reply:
Thanks! I will send a PR to fix this later.